DirectProgramming/DPC++FPGA/ReferenceDesigns/qri/src/CMakeLists.txt

```diff
@@ -0,0 +1,119 @@
+set(TARGET_NAME qri)
+set(HOST_SOURCE_FILE qri_demo.cpp)
+set(EMULATOR_TARGET ${TARGET_NAME}.fpga_emu)
+set(FPGA_TARGET ${TARGET_NAME}.fpga)
+set(FPGA_EARLY_IMAGE ${TARGET_NAME}_report.a)
+
+# FPGA board selection
+if(NOT DEFINED FPGA_BOARD)
+    set(FPGA_BOARD "intel_a10gx_pac:pac_a10")
+    message(STATUS "FPGA_BOARD was not specified.\
+                    \nConfiguring the design to run on the default FPGA board ${FPGA_BOARD} (Intel(R) PAC with Intel Arria(R) 10 GX FPGA). \
+                    \nPlease refer to the README for information on board selection.")
+else()
+    message(STATUS "Configuring the design to run on FPGA board ${FPGA_BOARD}")
+endif()
+
+# This is a Windows-specific flag that enables error handling in host code
+if(WIN32)
+    set(PLATFORM_SPECIFIC_FLAGS "/EHsc /Qactypes /Wall")
+else()
+    set(PLATFORM_SPECIFIC_FLAGS "-qactypes -Wall")
+endif()
+
+# Set design parameters according to the selected board
```

Comment:
Take a look at Mike's recent CMake changes (https://github.com/oneapi-src/oneAPI-samples/pull/817); may as well integrate them into both your QRD and QRI.